### PR TITLE
add version list and mult-version key doc

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

File: content/en/docs/overview/singleton-service/singleton-service-apis.md [hunk 1]
```diff
@@ -573,9 +573,160 @@ The value of 2 returns the formatted pattern data according to the provided lang
...
 
+#### **GET**/i18n/api/v2/translation/products/{productName}/versionlist
 
+**Parameters:**
+
+productName: **(required)** (String), to indicate the product name registered in Singleton; e.g. Testing
+
+**Return value:**
+
+(JSON Object) The request productName's all version list.
```
Comment (linr211) [inline content/en/docs/overview/singleton-service/singleton-service-apis.md:587]: it confuse me that JSON Object's is list.

File: content/en/docs/overview/singleton-service/singleton-service-apis.md [hunk 2]
```diff
@@ -573,9 +573,160 @@ The value of 2 returns the formatted pattern data according to the provided lang
...
+```
+{
+ "response":{"code":200,"message":"OK","serverTime":""},
+ "signature":"",
+ "data":{"versions":["1.0.0"],"productName":"MULTCOMP"}
+}
+```
+
+# **API - Get Mult-version Key translation**
+The API is used to get multi-version key's translation
```
Comment (linr211) [inline content/en/docs/overview/singleton-service/singleton-service-apis.md:601]: it's not readable to me about 'multi-version key'.

File: content/en/docs/overview/singleton-service/singleton-service-apis.md [hunk 3]
```diff
@@ -573,9 +573,160 @@ The value of 2 returns the formatted pattern data according to the provided lang
...
 
+locale: **(required)** (String), locale string
 
+component: **(required)** (String), component name
 
+key: **(required)** (String), an id value to identify translation 
 
+**Return value:**
+
+(JSON Object) According to mult-version key request parameter get translation list
```
Comment (linr211) [inline content/en/docs/overview/singleton-service/singleton-service-apis.md:619]: no readble

File: content/en/docs/overview/singleton-service/singleton-service-apis.md [hunk 4]
```diff
@@ -573,9 +573,160 @@ The value of 2 returns the formatted pattern data according to the provided lang
...
+      "translation": "this is a test11",
+      "locale": "en",
+      "key": "key.test1",
+      "component": "component1",
+      "status": "Source is not provided, the existing translation is found and returned"
+    }
+  ]
+}
+```
+Request versions string only part version key have translations
```
Comment (linr211) [inline content/en/docs/overview/singleton-service/singleton-service-apis.md:704]: not readable to me for 'Request versions string only part version key have translations'

File: content/en/docs/overview/singleton-service/singleton-service-apis.md [hunk 5]
```diff
@@ -573,9 +573,160 @@ The value of 2 returns the formatted pattern data according to the provided lang
...
+      "translation": "this is a test11",
+      "locale": "en",
+      "key": "key.test1",
+      "component": "component1",
+      "status": "Source is not provided, the existing translation is found and returned"
+    }
+  ]
+}
+```
+**{Singleton service}**/i18n/api/v2/translation/products/MULTCOMP/multiVersionKey?versions=all&locale=en&component=component1&key=key.test1
```
Comment (linr211) [inline content/en/docs/overview/singleton-service/singleton-service-apis.md:664]: Hardcoding the version=all looks not good to me, i would prefer to get all by versions=''

File: content/en/docs/overview/singleton-service/singleton-service-apis.md [hunk 6]
```diff
@@ -573,9 +573,160 @@ The value of 2 returns the formatted pattern data according to the provided lang
...
+# **API - Get Mult-version Key translation**
+The API is used to get multi-version key's translation
+
+#### **GET**/i18n/api/v2/translation/products/{productName}/multiVersionKey
+
+**Parameters:**
+
+productName: **(required)** (String), to indicate the product name registered in Singleton; e.g. Testing
+
+versions: **(required)** (String), a string contains multiple translation version, separated by commas,(if versions string is ‘all’, It will get all version translation)
```
Comment (linr211) [inline content/en/docs/overview/singleton-service/singleton-service-apis.md:609]: 1. typo: no , before (
2. why not use versions=[a, b, c] to represent a list, looks like separating the string by comma is not standard.

File: content/en/docs/overview/singleton-service/singleton-service-apis.md (hunk 1 above)
Comment (tigershi) [reply content/en/docs/overview/singleton-service/singleton-service-apis.md:587]: done

File: content/en/docs/overview/singleton-service/singleton-service-apis.md (hunk 2 above)
Comment (tigershi) [reply content/en/docs/overview/singleton-service/singleton-service-apis.md:601]: done

File: content/en/docs/overview/singleton-service/singleton-service-apis.md (hunk 3 above)
Comment (tigershi) [reply content/en/docs/overview/singleton-service/singleton-service-apis.md]: done

File: content/en/docs/overview/singleton-service/singleton-service-apis.md (hunk 5 above)
Comment (tigershi) [reply content/en/docs/overview/singleton-service/singleton-service-apis.md:664]: It is the custom requirement

File: content/en/docs/overview/singleton-service/singleton-service-apis.md (hunk 6 above)
Comment (tigershi) [reply content/en/docs/overview/singleton-service/singleton-service-apis.md]: all API use the "a,b,c" to represent a list

File: content/en/docs/overview/singleton-service/singleton-service-apis.md (hunk 4 above)
Comment (tigershi) [reply content/en/docs/overview/singleton-service/singleton-service-apis.md]: done

**Review by jin-jasper — APPROVED** [review]

**Review by nannany00 — APPROVED** [review]

**Pull request merged**